presto-main/src/main/java/com/facebook/presto/resourcemanager/DistributedClusterStatsResource.java

```diff
@@ -56,7 +57,7 @@ public DistributedClusterStatsResource(NodeSchedulerConfig nodeSchedulerConfig,
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    public Response getClusterStats()
+    public Response getClusterStats(@QueryParam("adjustedQueueSize") long adjustedQueueSize)
```

Comment:
I think we talked about this in detail, but wanted to put it here just to keep history. Per our discussion we have both tried to figure if we can move this to resource manager. But overall, it is hard to refactor as with the current design (based on our conversation with Abhishek). @tdcmeehan could we have some discussion around whether can we move this to ResourceManager.

Reply:
I was a bit hesitant as well to put this as a parameter. I think ideal way would be compute this on RM end if having the adjusted queue size calculation on RM side makes sense.

Reply:
I have added a TODO comment here to avoid passing this as parameter. Once we finalize an approach to refactor code to calculate adjusted queue size on RM, I will have another PR out to address the TODO. This is to unblock @basarhamdionat  as one of his features require this change urgently.

Reply:
Debating whether leaving this is better or passing 0 as value for resource manager? As the code is not used besides of this gtr API. But if @swapsmagic or @tdcmeehan thinks this is better, I am OK.

Reply:
Based on discussion, I have added 0 for RM side of calculation. We will have a separate PR to compute adjusted queue size on RM end.